### PR TITLE
[FEATURE] Fileinfo: more info

### PR DESCRIPTION
**Pull request by cbielow**
* extend mslevel + fragmentation method information 

also fixes a test on Apple Clang (amends #4838)

### CODE REVIEW

File: src/tests/topp/FileInfo_5_output.txt [hunk 1]
```diff
@@ -13,15 +13,15 @@ Total number of peaks: 3149
 Number of spectra: 10
 
 Ranges:
-  retention time: 20.87 .. 39.97
+  retention time: 20.87 .. 39.97 sec (0.3 min)
```
Comment (timosachsenberg) [inline src/tests/topp/FileInfo_5_output.txt:16]: start also in min?

**Comment by timosachsenberg** [comment]
feel free to merge

File: src/tests/topp/FileInfo_5_output.txt (hunk 1 above)
Comment (timosachsenberg) [reply src/tests/topp/FileInfo_5_output.txt:16]: maybe as 
```
(0.2 min .. 0.3 min)
```

Comment (timosachsenberg) [reply src/tests/topp/FileInfo_5_output.txt:16]: ah ok it's already the duration of the whole run

Comment (cbielow) [reply src/tests/topp/FileInfo_5_output.txt:16]: should I use `(span 30 min)` to make it more clear?

**Pull request merged**